cmd/deploy.go

```diff
@@ -390,9 +390,7 @@ func (dc *deployCmd) run() error {
 		os.Exit(1)
 	}
 
-	template, parameters, err := templateGenerator.GenerateTemplate(dc.containerService, engine.DefaultGeneratorCode, BuildTag)
```

Comment:
Should we also delete the old code/templates so they don't get modified in future PRs by mistake? If we need to revert this change, it will be to revert everything together.